src/content/en/feedback/file-a-bug/index.md

```diff
@@ -0,0 +1,60 @@
+project_path: /web/_project.yaml
+book_path: /web/feedback/_book.yaml
+
+{# wf_updated_on: 2016-10-20 #}
+{# wf_published_on:  #}
+
+# How to file a good bug {: .page-title }
+
```

Comment:
Dru and @foolip (https://github.com/foolip) are working on that document, so I'd redirect questions their way.